howdy/src/cli/disable.py

```diff
@@ -35,13 +35,13 @@
 	sys.exit(1)
 
 # Don't do anything when the state is already the requested one
-if out_value == config.get("core", "disabled"):
+if out_value == config.get("core", "disabled", fallback=False):
```

Comment:
I would like the fallbacks to be as safe as possible, would it be okay for your implementation if this defaulted to to true? Then howdy would be disabled automatically

Reply:
Sure.